src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx

```diff
@@ -24,6 +25,7 @@ const ProposedSolutionsList: React.FC<ProposedSolutionsListProps> = ({ proposedS
   const [showEditableForm, setShowEditableForm] = useState<boolean>(false);
   const auth = useAuth();
   const { isLoading, isError, error, mutateAsync } = useCreateProposeSolution();
+  const toast = useToast();
```

Comment:
the tests for these two files fail bc this useToast() hook relies on a context that we wrap the whole app in called `<ToastProvider> </ToastProvider>`. This context works by basically adding a toast next to whatever content is a child of the toast provider (in this case our whole app). However, in the tests, we render components individually, so the component isn't wrapped in this context provider (hence the error we're getting). So, you need to wrap the component inside the renderComponent methods in both of these tests with a ToastProvider component to fix the problem